_includes/footer.html

```diff
@@ -15,10 +15,10 @@
             <div class="grid-row padding-bottom-1">
                 <div class="grid-col-auto site-footer__links-container">
                     <ul class="grid-row grid-gap site-footer__links--datagov">
-                        <li><a href="{{ '/' | url }}about">About</a></li>
-                        <li><a href="{{ '/' | url }}open-gov/">Open Government</a></li>
-                        <li><a href="{{ '/' | url }}privacy-policy">Website Policies</a></li>
-                        <li><a href="//www.performance.gov/">PERFORMANCE.GOV</a></li>
+                        <li><a href="{% page '/about/' %}">About</a></li>
+                        <li><a href="{% page '/open-gov/' %}">Open Government</a></li>
+                        <li><a href="{% page '/privacy-policy/' %}">Website Policies</a></li>
+                        <li><a href="{% link 'https://www.performance.gov/' %}">PERFORMANCE.GOV</a></li>
```

Comment:
Is the link function even necessary here?

Reply:
Not technically.